src/containers/ap/ApGuestRegister.tsx

```diff
@@ -0,0 +1,316 @@
+				</td>
+				</tr>
+				<tr>
+				<td>
+				<DateTriPicker<Form, DateTriPickerProps<Form>>
+				years={years}
+				monthID="dobMonth"
+				dayID="dobDay"
+				yearID="dobYear"
+				isRequired={true}
+				monthValue={self.state.formData.dobMonth}
+				dayValue={self.state.formData.dobDay}
+				yearValue={self.state.formData.dobYear}
+				updateAction={updateState}
+				/>
+				</td>
+				<td>
+				<PhoneTriBox<Form,  PhoneTriBoxProps<Form>>
+				label="Phone"
+				firstID="guestPhoneFirst"
+				secondID="guestPhoneSecond"
+				thirdID="guestPhoneThird"
+				extID="guestPhoneExt"
+				typeID="guestPhoneType"
+				firstValue={self.state.formData.guestPhoneFirst}
+				secondValue={self.state.formData.guestPhoneSecond}
+				thirdValue={self.state.formData.guestPhoneThird}
+				extValue={self.state.formData.guestPhoneExt}
+				typeValue={self.state.formData.guestPhoneType}
+				updateAction={updateState}
+				isRequired={true}
+			/>
+				</td>
+				</tr>
+				<tr>
+				<td>
+				<FormInput
+				id="email"
+				label="Email"
+				isPassword={false}
+				isRequired
+				value={self.state.formData.email}
+				updateAction={updateState}
+				/>
+				</td>
+				<td>
+				<p>opt out</p>
+				</td>
+				</tr>
+				<th>Emergency Contact Information</th>
+				<tr>This should be someone close to you who is not going on the water with you.</tr>
+				<tr>For under 18 guests this should be a parent or guardian.</tr>
+				<tr>
+				<td>
+				<FormInput
+				id="ecFirstName"
+				label="First Name"
+				isPassword={false}
+				isRequired
+				value={self.state.formData.ecFirstName}
+				updateAction={updateState}
+				/>
+				</td>
+				<td>
+				<FormInput
+				id="ecLastName"
+				label="Last Name"
+				isPassword={false}
+				isRequired
+				value={self.state.formData.ecLastName}
+				updateAction={updateState}
+				/>
+				</td>
+				</tr>
+				<tr>
+				<td>
+				<FormInput
+				id="ecRelationship"
+				label="Relationship"
+				isPassword={false}
+				isRequired
+				value={self.state.formData.ecRelationship}
+				updateAction={updateState}
+				/>
+				</td>
+				<td>
+				<PhoneTriBox<Form,  PhoneTriBoxProps<Form>>
+				label="Emergency Contact Phone"
+				firstID="ecPhoneFirst"
+				secondID="ecPhoneSecond"
+				thirdID="ecPhoneThird"
+				extID="ecPhoneExt"
+				typeID="ecPhoneType"
+				firstValue={self.state.formData.ecPhoneFirst}
+				secondValue={self.state.formData.ecPhoneSecond}
+				thirdValue={self.state.formData.ecPhoneThird}
+				extValue={self.state.formData.ecPhoneExt}
+				typeValue={self.state.formData.ecPhoneType}
+				updateAction={updateState}
+				isRequired={true}
+				/>
+				</td>
+				</tr>
+				<tr>
+				<td>
+				</td>	
+				<td>
+				{ progressButton }
+				</td>
+				</tr>
+			</tbody></table>
+		</div>);
+
+		const adultWaiverContent = (<div id="adultwaiver">
+		<table width="100%"><tbody>
+				<tr>
+					<iframe title="Waiver of Liability" src="../../../waivers/live/ApGuestWaiver.html" width="100%" height="400px"></iframe>
```

Comment:
Change this to an absolute path, otherwise if we restructure the paths of the actual pages, it might not be exactly three levels below root anymore.  /public will always be at the URL root.  I'm tempted to say it should be turned into a PathWrapper but for a public thing it's probably fine without

```suggestion
					<iframe title="Waiver of Liability" src="/waivers/live/ApGuestWaiver.html" width="100%" height="400px"></iframe>
```